File: conformance/send_event.mjs
// Conformance sender: the REAL @sentry/node SDK, unmodified, DSN-only.
import * as Sentry from "@sentry/node";

const dsn = process.argv[2];

Sentry.init({
  dsn,
  release: "conformance-node-1.0",
  environment: "conformance",
  defaultIntegrations: false,
});

function loadProfile() {
  const user = undefined;
  return user.name; // TypeError
}

try {
  loadProfile();
} catch (err) {
  Sentry.captureException(err);
}

await Sentry.flush(10000);
console.log("node-sdk: flushed");

File: conformance/send_event.py
"""Conformance sender: the REAL sentry-sdk (Python), unmodified, pointed at
leser by DSN alone (order.md §3). Sends one handled exception and flushes."""
import sys

import sentry_sdk

dsn = sys.argv[1]

sentry_sdk.init(
    dsn=dsn,
    release="conformance-py-1.0",
    environment="conformance",
    default_integrations=False,  # deterministic: no auto-instrumentation noise
)

def checkout_cart():
    prices = {"apple": 3}
    return prices["banana"]  # KeyError

try:
    checkout_cart()
except KeyError:
    sentry_sdk.capture_exception()

ok = sentry_sdk.flush(timeout=10)
print("python-sdk: flushed")
